### ISSUE TITLE
AniDB not scraping anything

### ISSUE THREAD

**Issue by thaleb** [tinyMediaManager/tinyMediaManager#475]
__What TMM version are you using?__
tmm_2.9.16_42413ce_win

__release, pre-release, nightly, or directly from GitHub/branch?__
release

__What is the actual behaviour?__
I try to scrape any anime with aniDB but tmm will not find the series. Other scrapers work but anidb doesn't do anything.

__What is the expected behaviour?__
That the anime is found using the aniDB scraper.

__Steps to reproduce:__
Start tmm with gui
Go to TV shows
Try to scrape any anime series with the aniDB scraper

__Additional__
Windows 10
I added the tmm.log
[tmm.log](https://github.com/tinyMediaManager/tinyMediaManager/files/3150208/tmm.log)

**Comment by myron0815** [comment]
Yes, it seems, they finally switched/corrected their webserver to deliver GZipped content.
We might get this into next v2... already fixed that in v3...

**Comment by thaleb** [comment]
> already fixed that in v3...

I tried using the nightly build last night (3.0 SNAPSHOT) and got a IP ban at anidb. I just started up TMM, went trough the initial setup telling tmm which folder contains what. Then I let initialize all the files which took a little while. After tmm finished initializing i tried to scrape one anime...which did not work. When looking at the log I only saw a connection timed out while getting GZ file. I tried to access the anidb.net site and got the same connection timed out error. Seemed i gotten a IP ban because tmm tried to download the titel dump multiple times which is not allowed....

I posted on the anidb forum why i had gotten a IP ban and got the following reply from a mod:

>  you were banned because the software you used did download the title dump more than once per day. in face it went over the limit way further than that
as specified in https://wiki.anidb.net/w/API#Data_Dumps "YOU DO NOT REQUEST THIS FILE MORE THAN ONCE PER DAY"
automatic 
the title dump is only updated _once_ per day. there is no reason to download more than that. the software should cache it and at most download again after 24h. violations against this will lead to bans as you have seen.


Source: https://anidb.net/perl-bin/animedb.pl?show=cmt&id=89531

>
